image/pubcloud/product/sles_sap/config.kiwi

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="6.8" name="SLES15-SAP-Migration">
+    <description type="system">
+        <author>Public Cloud Team</author>
+        <contact>public-cloud-dev@susecloud.net</contact>
+        <specification>
+            SLES15 Distribution Migration Live System to upgrade
+            SLES12-SP4+ SAP instances to SLES15 SP3
```

Comment:
We should move to 15 SP4 as we are withing 6 months of SP3 reaching EOL and it will take us probably a couple of months to get this change through the system.